NickvisionMoney.GNOME/Views/MainWindow.cs

```diff
@@ -233,6 +235,7 @@ public MainWindow(MainWindowController controller, Adw.Application application)
         _btnNewAccount.AddCssClass("pill");
         _btnNewAccount.AddCssClass("suggested-action");
         _btnNewAccount.SetDetailedActionName("win.newAccount");
+        _btnNewAccount.SetMarginBottom(12);
```

Comment:
This will add space between buttons when they will be vertical. It's better to add margin to flowbox.